OpenDreamRuntime/Objects/Types/DreamList.cs

```diff
@@ -142,6 +142,14 @@ public virtual void AddValue(DreamValue value) {
         _values.Add(value);
     }
 
+    public virtual void AddValueRange(DreamList valueRange) {
+        _values.AddRange(valueRange.GetValues());
+    }
+
+    public virtual void AddValueRange(IEnumerable<DreamValue> valueRange) {
+        _values.AddRange(valueRange);
+    }
```

Comment:
Adding helper functions like this that perform directly on `_values` should be avoided. These would have to be overridden on every DreamList type to handle their special behaviors.